src/store/data.js

```diff
@@ -158,13 +162,24 @@ export const useDataStore = defineStore('data', {
 				return false
 			}
 
+			res.data.forEach(row => {
+				this.rowMetadata[row.id] = {
```

Comment:
Could this lead to problems if we load a context that has two views that show the same row at the same time? Then we could have two different viewIds that are used